lib/trento_web/controllers/page_html/index.html.heex

```diff
@@ -1,4 +1,8 @@
-<div id="trento" />
+<div
```

Comment:
Why don't we put the `version` in the `config`?
It is not a config, that's true, but it would enable to use the `getFromConfig` function.
And after all, the value is exposed in both cases.

Devils advocate question: does exposing the server version to not logged users expose some "you need to be logged" info?